deeplabcut/generate_training_dataset/trainingsetmanipulation.py

```diff
@@ -728,6 +731,7 @@ def create_training_dataset(
     net_type=None,
     augmenter_type=None,
     posecfg_template=None,
+    superanimal_name = ""
```

Comment:
just a bit of styling to follow the rest of the file

```suggestion
    superanimal_name="",
```